<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>566</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#1d1d1d;
}

QMenuBar{
background-color:#1d1d1d;
padding:5px;
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}

QMenuBar::item{
background-color:#1d1d1d;
color:#fff;
padding:5px;

}

QMenu{
color:#fff;
padding:0;
}

QMenu::item:selected{
color:#fff;
background-color:#00aba9;
}

QTableWidget{
background-color:#3d3d3d;
color:#fff;
  selection-background-color: #da532c;
border:solid;
border-width:3px;
border-color:#da532c;
}
QHeaderView::section{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));
border:none;
border-top-style:solid;
border-width:1px;
border-top-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));
color:#fff;

}
QHeaderView{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(20, 158, 217, 255), stop:1 rgba(36, 158, 217, 255));

border:none;
border-top-style:solid;
border-width:1px;
border-top-color:#149ED9;
color:#fff;
	font: 75 12pt &quot;Calibri&quot;;
}

QTableCornerButton::section{
border:none;
background-color:#149ED9;
}

QListWidget{
background-color:#3d3d3d;
color:#fff;
}

QMenu{
background-color:#3d3d3d;
}
QStatusBar{
background-color:#7e3878;
color:#fff;
}

QPushButton{
border-style:solid;

background-color:#3d3d3d;
color:#fff;
border-radius:7px;
}
QPushButton:hover{
color:#ccc;
	background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45, 45, 45, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
	border-color:#2d89ef;
border-width:2px;
}

QPushButton:pressed{
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29, 29, 29, 255), stop:0.505682 rgba(45, 45, 45, 255), stop:1 rgba(29, 29, 29, 255));
}


QTabWidget::tab{
background-color:#3d3d3d;
}

QLineEdit{
border-radius:0;
}

QProgressBar{
border-radius:0;
text-align:center;
color:#fff;
background-color:transparent;
border: 2px solid #e3a21a;
border-radius:7px;
	font: 75 12pt &quot;Open Sans&quot;;

}

QProgressBar::chunk{
background-color:#2d89ef;
width:20px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="toolsWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>6</number>
        </property>
        <item row="6" column="0" colspan="3">
         <widget class="VEEllipseSettings" name="ellipseSettings" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="3">
         <widget class="polyline" name="polylineSettings" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="rectangle" name="rectangleSettings" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="toolButton_bezier">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/path.png</normaloff>:/icons/path.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="toolButton_ellipse">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>椭圆</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/椭圆.png</normaloff>:/icons/椭圆.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QToolButton" name="toolButton_Home">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/up.png</normaloff>:/icons/up.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QToolButton" name="toolButton_text">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/text.png</normaloff>:/icons/text.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="butRectangle">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/rectangle.png</normaloff>:/icons/rectangle.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="toolButton_ZoomOut">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/缩小1.png</normaloff>:/icons/缩小1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="butSave">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/文件保存.png</normaloff>:/icons/文件保存.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QToolButton" name="toolButton_SaveAs">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/文件另存1.png</normaloff>:/icons/文件另存1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="toolButton_ZoomIn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/放大1.png</normaloff>:/icons/放大1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="butDefault">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/select.png</normaloff>:/icons/select.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="toolButton_clear">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/删除操作1.png</normaloff>:/icons/删除操作1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="butLine">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/直线.png</normaloff>:/icons/直线.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>31</width>
            <height>31</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="VEWorkplaceView" name="workplaceView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777195</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/scene.ico</normaloff>:/icons/scene.ico</iconset>
   </property>
   <property name="text">
    <string>Readme</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/saveAs.png</normaloff>:/icons/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/zoomIn.png</normaloff>:/icons/zoomIn.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/zoomOut.png</normaloff>:/icons/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionDeveloper">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/工具.png</normaloff>:/icons/工具.png</iconset>
   </property>
   <property name="text">
    <string>Developer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>rectangle</class>
   <extends>QWidget</extends>
   <header>rectangle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>polyline</class>
   <extends>QWidget</extends>
   <header>polyline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VEEllipseSettings</class>
   <extends>QWidget</extends>
   <header>ellipse.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VEWorkplaceView</class>
   <extends>QGraphicsView</extends>
   <header>workplaceview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
